samples/52.teams-messaging-extensions-search-auth-config/src/main/java/com/microsoft/bot/sample/teamssearchauth/TeamsMessagingExtensionsSearchAuthConfigBot.java

```diff
@@ -23,6 +23,7 @@
 import org.apache.commons.lang3.StringUtils;
 import org.slf4j.LoggerFactory;
 
+import javax.smartcardio.Card;
```

Comment:
Is this ok? It's weird that with the change of DBI -> SI we need to import new things